packages/syft/src/syft/service/project/distributed_project.py

```diff
@@ -166,17 +166,24 @@ def request_execution(self, blocking: bool = True) -> Any:
             print(assets_transferred.message)
 
         result_parts = []
+        errors = []
         for client in self.clients.values():
             result = client.api.services.enclave.request_execution(
                 service_func_name=self.code.service_func_name
             )
             if isinstance(result, SyftError):
-                raise SyftException(result.message)
-            result_parts.append(result)
+                errors.append(result.message)
```

Comment:
We can also return early as soon as there is an error. This pattern will be more consistent with the rest of the codebase.